robot/basestation/static/js/app.js

```diff
@@ -1,5 +1,6 @@
 PING_THROTTLE_TIME = 1000
-ERROR_SOUND_PATH = '../../static/sound/error.mp3'
+ERROR_SOUND_PATH = '../../static/sound/error.wav'
```

Comment:
I feel like it would make more sense to call this the warning sound.
Is this sound used for actual errors in some places?

Reply:
Yes, this is the sound triggered when the voltage or temperature is in danger zone (red text).

Reply:
@AFknUsername There's two levels of notification for the voltage. 

We have a zone that if the voltage is under that value, we want to immediately cease operations because it could damage the rover. Hence, when this value is reached there's an error sounds and a pop up model dialogue. The operator is expected to hit the killswitch of the rover. This is bad though, because we never actually want to enter this danger zone.

This issue was created to notify the operator during a warning zone (not in the error zone but close). The warning sends a light notification (not a popup, but a top-right chrome-styled notification). It is lightweight so it doesnt distract the operator in a critical moment, but still visible. AFAIK it does not have a sound. So the sound is for when we are in the danger zone. @AFknUsername

Reply:
I see that makes much more sense, thanks